git_trac/config.py

```diff
@@ -73,7 +73,7 @@ def username(self):
         try:
             return self._load('trac.username')
         except GitError:
-            raise SystemExit('Use "git trac config --user=<name>"'
+            raise ValueError('Use "git trac config --user=<name>"'
```

Comment:
Can you add a new exception class (e.g. AuthenticationError) instead of using ValueError?